.github/workflows/e2e-fork.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         env:
           TEST_EXCLUSIONS: "TestInterTxTestSuite,TestIncentivizedInterTxTestSuite,TestUpgradeTestSuite"
 
-  e2e:
+  e2e-fork:
```

Comment:
Changing the name of this job would allow us to add it separately in the branch protection rules and would help from a UI perspective somewhat. Note that renaming a job doesn't make it visible as an option when selecting required checks until it has been executed at least once (i.e, Github doesn't parse the yaml in your `.github` folder to discover them but looks at what jobs get executed when one is launched).